Header header
string gesture_type  # "pinch", "fist", "open", "peace", "thumbs_up"
float32 confidence
